demo/api/package.json

```diff
@@ -63,7 +66,8 @@
         "rimraf": "^6.0.1",
         "rxjs": "^7.8.2",
         "slugify": "^1.6.6",
-        "uuid": "^11.1.0"
+        "uuid": "^11.1.0",
+        "react-intl": "^6.0.0"
```

Comment:
Do we need react-intl in the API? Can't we use [@formatjs/intl](https://formatjs.github.io/docs/intl)?

Reply:
Honestly I don't know. But we will use translation in the html used for mails which is probably generated with react--components... does this matter?